src/west/app/project.py

```diff
@@ -155,19 +156,49 @@ def __init__(self):
             'init',
             'create a west workspace',
             f'''\
-Creates a west workspace.
+Initialize a west workspace.
+
+West supports two ways of creating a workspace:
 
-With -l, creates a workspace around an existing local repository;
-without -l, creates a workspace by cloning a manifest repository
-by URL.
+1. Remote Manifest Repositories
+----------------------------
+If `-m` is given, west clones the manifest repository from the specified URL
+into the workspace. Config option manifest.path will point at this cloned
+directory.
 
-With -m, clones the repository at that URL and uses it as the
-manifest repository. If --mr is not given, the remote's default
-branch will be used, if it exists.
+- If `-m` is not provided, west defaults to the Zephyr manifest repository:
+  {MANIFEST_URL_DEFAULT}.
+- With `--mr`, you can specify the revision (branch, tag, or sha) of the
+  manifest repository.
+- If `--mr` is omitted, the repository's default branch (if available) is used.
 
-With neither, -m {MANIFEST_URL_DEFAULT} is assumed.
+Note: The west workspace will be created in the current working directory, if
```

Comment:
```suggestion
Note: The west workspace will be the current working directory, if
```

I think this help text needs to entirely ban the expression "workspace in X/" because this expression is ambiguous: it can mean either `X/.west/` or `X/Y/.west`



This is even more important with this new `--topdir` option.